### PR TITLE
Add initial Tox configuration and a couple of fixes

### PR DESCRIPTION
**Pull request by tjanez**
It would be great if we could include this before the next release.

At least the commit that ensures all source files and supplementary package data are in source distribution.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,8 +30,16 @@
...
+                'descriptors/*.yml',
+                'processes/**/*.yml',
+                'tests/files/*',
+                'tools/*.py',
+                'tools/*.R',
+            ]
+        },
         zip_safe=False,
         install_requires=(
             "resolwe>=1.0.0",
```
Comment (mstajdohar) [inline setup.py]: Use single quotes.

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -4,14 +4,26 @@ Change Log
...
+-------
+* Removed obsolete utilities superseded by resolwe-runtime-utils
+
+Fixed
+-----
+* Included all source files and supplementary package data in sdist
+
 Added
 -----
 * "flow_collection: sample" to processes
```
Comment (mstajdohar) [inline docs/CHANGELOG.rst]: Maybe also fix this to ``, since you're at it.

File: setup.py (hunk 1 above)
Comment (tjanez) [inline setup.py]: Hmm, I'm using single quotes?

Do you want me to change all double-quotes in `setup.py` to single quotes? If yes, I would rather do it in a separate commit.

Comment (mstajdohar) [inline setup.py]: "resolwe>=1.0.0"

Comment (mstajdohar) [inline setup.py]: Oh, I see. Not marked green, sorry.

Comment (mstajdohar) [inline setup.py]: So, the general rule is that text printed to the user should be in double quotes and the other in single. You can fix the whole `setup.py` if you wish, but not urgent.

**Comment by mstajdohar** [comment]
LGTM

File: docs/CHANGELOG.rst (hunk 2 above)
Comment (tjanez) [inline docs/CHANGELOG.rst]: I would rather do it tomorrow when we'll prepare a new release and as part of preparing the release, we will also clean up/prettify the CHANGELOG.

The reason I'm against it is that I don't want to fix unrelated things in commits. Making a separate commit for this change seems overkill if the release is so close.

File: setup.py (hunk 1 above)
Comment (tjanez) [inline setup.py]: Ok, will do.

**Comment by mstajdohar** [comment]
LGTM
